tox.ini

```diff
@@ -34,6 +34,7 @@ basepython =
     3.5: python3.5
     3.6,apicheck,pydocstyle: python3.6
     3.7: python3.7
+    3.8: python3.8
```

Comment:
is this released yet?

Reply:
Python 3.8 is not yet released. Beta 3 was released recently. Looks like they're [targeting October 21, 2019](https://www.python.org/dev/peps/pep-0569/#schedule).

Reply:
Exactly. We are updating to [Python 3.8 in Fedora 32](https://fedoraproject.org/wiki/Changes/Python3.8) already.

Reply:
remove this. we will need this in Travis as well

Reply:
Done.